### PR TITLE
Update explorer-backend schemas to 1.16.0

### PR DESCRIPTION
**Pull request by tdroxler**
This adds all metadatas, for tokens/nfts

Required by FE team for the next update of the explorer

### CODE REVIEW

File: packages/web3/src/api/api-explorer.ts [hunk 1]
```diff
@@ -136,6 +136,15 @@ export interface ExplorerInfo {
   lastFinalizedInputTime: number
 }
 
+export interface FungibleTokenMetadata {
+  /** @format 32-byte-hash */
+  id: string
+  symbol: string
+  name: string
+  /** @format uint256 */
+  decimals: string
```
Comment (nop33) [inline packages/web3/src/api/api-explorer.ts:145]: 😢

Comment (tdroxler) [reply packages/web3/src/api/api-explorer.ts:145]: yeah I know, but as we discussed it's safer for the backend to return a BigNumer as that's what we get from the node.

Comment (nop33) [reply packages/web3/src/api/api-explorer.ts:145]: I know, I know, I'm still sad tho :P

Comment (nop33) [reply packages/web3/src/api/api-explorer.ts:145]: I'll get over it, don't worry 😆

**Review by polarker — APPROVED** [review]
LGTM

**Pull request merged**